yarn.lock

```diff
@@ -331,10 +331,9 @@
     minimatch "^3.0.4"
     strip-json-comments "^3.1.1"
 
-"@github/template-parts@^0.4.0":
+"@github/template-parts@git+https://github.com/muxinc/template-parts.git#mux-player":
   version "0.4.0"
-  resolved "https://registry.yarnpkg.com/@github/template-parts/-/template-parts-0.4.0.tgz#2dc7ebc511650913a8c3962f374b541eea45693f"
-  integrity sha512-7jMaJAnjvqpSEiegDkAtbjV9fD+kyZP8TAc2la1w5dwO3VaSg5/HULcn9XQxLqMli8pYcJxtY820HTqFwlqleQ==
+  resolved "git+https://github.com/muxinc/template-parts.git#b6f556fb97afe5ddade36ac5dee54ad683c009ff"
```

Comment:
This seems like it should have also come from yarnpkg.com?

Reply:
we're currently using a forked version while awaiting a fix to merge into mainline.